test/functionalTest/cases/0979_patch_v2_entities_eid/patch_v2_entities_eid.test

```diff
@@ -35,7 +35,8 @@ brokerStart CB
 # 03. GET /v2/entities/E1
 # 04. PATCH /v2/entities/E1 attr1=4
 # 05. GET /v2/entities/E1 - see attr1=4 AND attr2=2 (untouched)
-# 06. PATCH /v2/entities/E1 attr3=6 - error, as attr3 doesn't exist
+# 06. PATCH /v2/entities/E1 attr3=6
```

Comment:
I think this should be rolled-back... PATCH cannot be used to add new attributes, so if attr3 doesn't previously exist, then it should lead to an error.